tests/unit/test_solvers/test_casadi_algebraic_solver.py

```diff
@@ -174,157 +172,6 @@ def test_solve_with_input(self):
         np.testing.assert_array_equal(solution.y, -7)
 
 
-class TestCasadiAlgebraicSolverSensitivity(unittest.TestCase):
```

Comment:
why was this deleted?

Reply:
that was a mistake on my part, I'll put it back in